### ISSUE TITLE
Failed to build arb_utils:arb_utils

### ISSUE THREAD

**Issue by Albert221** [Rodsevich/arb_utils#25]
Hello, I can't install `arb_utils` CLI tool, after `dart pub cache clean`:

```
$ dart --version
Dart SDK version: 3.3.0 (stable) (Tue Feb 13 10:25:19 2024 +0000) on "macos_arm64"
```

```
$ dart pub global activate arb_utils
+ _fe_analyzer_shared 67.0.0.6s)
+ analyzer 6.4.1
+ arb_utils 0.7.1
+ archive 3.4.10
+ args 2.4.2
+ async 2.11.0
+ characters 1.3.0
+ chunked_stream 1.4.2
+ circular_buffer 0.11.0
+ clock 1.1.1
+ collection 1.18.0
+ convert 3.1.1
+ crypto 3.0.3
+ csv 5.1.1 (6.0.0 available)
+ dart_console2 3.0.0
+ dart_style 2.3.6
+ dcli 4.0.1-alpha.4 (4.0.1-alpha.13 available)
+ dcli_core 4.0.1-alpha.4 (4.0.1-alpha.11 available)
+ equatable 2.0.5
+ ffi 2.1.2
+ file 7.0.0
+ fixnum 1.1.0
+ flutter 0.0.0 from sdk flutter
+ flutter_web_plugins 0.0.0 from sdk flutter
+ functional_data 1.1.1
+ glob 2.1.2
+ globbing 1.0.0
+ http 1.2.1
+ http_parser 4.0.2
+ ini 2.1.0
+ intl 0.18.1 (0.19.0 available)
+ intl_utils 2.8.7
+ js 0.7.1
+ json2yaml 3.0.1
+ lists 1.0.1
+ logging 1.2.0
+ material_color_utilities 0.8.0 (0.11.1 available)
+ meta 1.11.0 (1.12.0 available)
+ mime 1.0.5
+ package_config 2.1.0
+ path 1.9.0
+ path_provider_linux 2.2.1
+ path_provider_platform_interface 2.1.2
+ path_provider_windows 2.2.1
+ petitparser 6.0.2
+ platform 3.1.4
+ plugin_platform_interface 2.1.8
+ pointycastle 3.7.4
+ posix 6.0.1
+ pub_semver 2.1.4
+ pubspec_lock 3.0.2
+ pubspec_manager 1.0.0
+ scope 4.1.0
+ settings_yaml 8.1.0-alpha.2
+ shared_preferences 2.2.2
+ shared_preferences_android 2.2.1
+ shared_preferences_foundation 2.3.5
+ shared_preferences_linux 2.3.2
+ shared_preferences_platform_interface 2.3.2
+ shared_preferences_web 2.3.0
+ shared_preferences_windows 2.3.2
+ sky_engine 0.0.99 from sdk flutter
+ source_span 1.10.0
+ sprintf 7.0.0
+ stack_trace 1.11.1
+ string_scanner 1.2.0
+ strings 3.1.1
+ sum_types 0.3.5
+ system_info2 4.0.0
+ term_glyph 1.2.1
+ typed_data 1.3.2
+ unicode 0.3.1
+ uuid 4.3.3
+ validators2 5.0.0
+ vector_math 2.1.4
+ watcher 1.1.0
+ web 0.5.1
+ win32 4.1.4 (5.3.0 available)
+ xdg_directories 1.0.4
+ yaml 3.1.2
Building package executables... (2.7s)
Failed to build arb_utils:arb_utils:
../../../.pub-cache/hosted/pub.dev/dcli-4.0.1-alpha.4/lib/src/script/dart_project.dart:189:38: Error: The type 'Dependencies' used in the 'for' loop must implement 'Iterable<dynamic>'.
 - 'Dependencies' is from 'package:pubspec_manager/src/pubspec/internal_parts.dart' ('../../../.pub-cache/hosted/pub.dev/pubspec_manager-1.0.0/lib/src/pubspec/internal_parts.dart').
 - 'Iterable' is from 'dart:core'.
    for (final dependency in pubSpec.dependencies) {
                                     ^
../../../.pub-cache/hosted/pub.dev/dcli-4.0.1-alpha.4/lib/src/script/dart_project_creator.dart:99:38: Error: The setter 'version' isn't defined for the class 'DependencyVersioned'.
 - 'DependencyVersioned' is from 'package:pubspec_manager/src/pubspec/dependency_versioned.dart' ('../../../.pub-cache/hosted/pub.dev/pubspec_manager-1.0.0/lib/src/pubspec/dependency_versioned.dart').
Try correcting the name to the name of an existing setter, or defining a setter or field named 'version'.
      (dcli! as DependencyVersioned).version = packageVersion;
                                     ^^^^^^^
../../../.pub-cache/hosted/pub.dev/dcli-4.0.1-alpha.4/lib/src/script/dart_project_creator.dart:107:42: Error: The setter 'version' isn't defined for the class 'DependencyVersioned'.
 - 'DependencyVersioned' is from 'package:pubspec_manager/src/pubspec/dependency_versioned.dart' ('../../../.pub-cache/hosted/pub.dev/pubspec_manager-1.0.0/lib/src/pubspec/dependency_versioned.dart').
Try correcting the name to the name of an existing setter, or defining a setter or field named 'version'.
      (dcliCore! as DependencyVersioned).version = packageVersion;
                                         ^^^^^^^
../../../.pub-cache/hosted/pub.dev/dcli-4.0.1-alpha.4/lib/src/script/dart_project_creator.dart:112:12: Error: The setter 'value' isn't defined for the class 'Name'.
 - 'Name' is from 'package:pubspec_manager/src/pubspec/internal_parts.dart' ('../../../.pub-cache/hosted/pub.dev/pubspec_manager-1.0.0/lib/src/pubspec/internal_parts.dart').
Try correcting the name to the name of an existing setter, or defining a setter or field named 'value'.
    ..name.value = _replaceInvalidCharactersForName(projectName)
           ^^^^^
```

**Comment by Albert221** [comment]
I wrote a quick tool that just provides you a binary to sort ARBs, to anyone concerned: https://github.com/leancodepl/arbsort

**Comment by Rodsevich** [comment]
fixed on #27
